Examples/Scripts/Python/material_validation.py

```diff
@@ -72,13 +93,23 @@ def runMaterialValidation(
     p.add_argument(
         "-m", "--map", type=str, default="", help="Input file for the material map"
```

Comment:
this could be an optional flag to avoid `if len(args.map) > 0` later

Reply:
How do I do that?

Reply:
```suggestion
        "-m", "--map", required=True, type=str, help="Input file for the material map"
```
later
```
if args.map is not None:
```

Reply:
I actually just removed the default, giving no map is also valid, e.g. for the GenericDetector or alike.